recursion/gnark-ffi/go/sp1/logging.go

```diff
@@ -0,0 +1,37 @@
+package sp1
+
+import (
+	"os"
+	"strings"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func init() {
+	// Get log level from environment variable
+	logLevel := strings.ToLower(os.Getenv("SP1_GO_LOG"))
+
+	// Configure the global log level based on the environment variable
+	switch logLevel {
+	case "disabled", "false", "none":
```

Comment:
To match the `RUST_LOG` names (from [env_logger](https://docs.rs/env_logger/latest/env_logger/#enabling-logging), which seems to be the same levels as [tracing](https://docs.rs/tracing/latest/tracing/struct.Level.html#filtering)), maybe we should add `"off"` to this list as well? Then setting both `RUST_LOG` and `SP1_GO_LOG` to `off` should disable logging for the application entirely
```suggestion
	case "disabled", "false", "none", "off":
```